deploy_zkevm_contracts.star

```diff
@@ -85,3 +94,12 @@ def run(plan, args):
             ]
         ),
     )
+
+
+def get_consensus_contract(args):
+    # Map data availability modes to consensus contracts.
+    consensus_contracts = {
+        "rollup": "PolygonZkEVMEtrog",
+        "validium": "PolygonValidiumEtrog",
+    }
+    return consensus_contracts.get(args["data_availability_mode"])
```

Comment:
Minor, but it seemed a little strange there are two functions that look almost identical but one is for the consensus and one is for the image. It seems a little cumbersome to change two functions in two very difficult locations if you want to add a new mode.

Certainly not a big deal though.

Reply:
True, let me refactor this quickly

Reply:
Changes addressed in [062579a](https://github.com/0xPolygon/kurtosis-cdk/pull/132/commits/062579a0da74165c35d1d4ad5d70468305c1c0c8)

All fn related to da have been moved to a specific da library. In addition, all occurrences of `validium` have been removed for cleanliness and now rely on an enum instead.

`validium` has been renamed `cdk-validium` so that we can easily support any other third party DA protocol like `avail-validium`, `celestia-validium` or `near-validium`.